holoviews/plotting/util.py

```diff
@@ -70,6 +72,19 @@ def collate(obj):
         raise Exception(undisplayable_info(obj))
 
 
+def initialize_dynamic(obj):
+    """
+    Initializes all DynamicMap objects contained by the object
+    """
+    dmaps = obj.traverse(lambda x: x, specs=[DynamicMap])
+    for dmap in dmaps:
+        if dmap.sampled:
+            # Skip initialization until plotting code
```

Comment:
Does this mean plotting does some of its own initialization? Maybe this utility could have a ``samples`` argument (or similar) to handle that case as well i.e if ``samples`` is None, this function stays the same as it is now, otherwise handles what is needed for sampled DynamicMaps.